README.md

```diff
@@ -12,30 +12,77 @@ These extensions allow you to test Vue component state and result markup togethe
 #### Create selectors for Vue components
 
 `VueSelector` allows you to select page elements by the component tagName or the nested component tagNames.
-For instance, you can create Vue selectors as follows
+
+Suppose you have the following markup.
 
 ```js
-VueSelector('list')
-VueSelector('list list-item')
-VueSelector() // returns the root Vue instance
+<div id="todo-app">
+    <todo-input />
+    <todo-list>
+        <todo-item priority="High">Item 1</todo-item>
+        <todo-item priority="Low">Item 2</todo-item>
+    </todo-list>   
+    <div className="items-count">Items count: <span>{{itemCount}}</span></div>
+</div>
+<script>
+    Vue.component('todo-input', {...});
+    Vue.component('todo-list', {...});
+    Vue.component('todo-item', {...});
+    
+    new Vue({ 
+        el:   '#todo-app',
+        data: {...}
+    });
+</script>
 ```
 
-You can combine Vue selectors with testcafe `Selector` filter functions like `.withText`, `.nth` and [other](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#functional-style-selectors).
+To get the root Vue instance use VueSelector constructor without parameters.
+```js
+import VueSelector from 'testcafe-vue-selectors';
+
+const rootVue = VueSelector();
+```
+Respectively, `rootVue` will contain `<div id="todo-app">`
```

Comment:
To get the root Vue instance, use the VueSelector constructor without parameters.
```js
import VueSelector from 'testcafe-vue-selectors';

const rootVue = VueSelector();
```
The `rootVue` variable will contain `<div id="todo-app">`